### PR TITLE
getLastBlockNumber rpc method

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: ui/eth_log_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package ui_test
+
+import (
+	"testing"
+
+	"github.com/privatix/dappctrl/ui"
+
```
Comment (dzeckelev) [inline ui/eth_log_test.go]: This line is not needed.

File: ui/eth_log.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package ui
+
+import "database/sql"
+
+// GetLastBlockNumber returns last known block number.
+func (h *Handler) GetLastBlockNumber(password string) (uint64, error) {
```
Comment (dzeckelev) [inline ui/eth_log.go]: Maybe `func (h *Handler) GetLastBlockNumber(password string) (*uint64, error)` or here a special case?

File: doc/ui/rpc.md [hunk 3]
```diff
@@ -137,6 +137,23 @@ This document describes a UI JSON RPC API located in "ui" namespace.
 *Result*: None.
 
 
+### Objects
+
+#### Get object
+
+*Method*:	`getObject`
+
+*Description*: Get an object of a specified type..
```
Comment (dzeckelev) [inline doc/ui/rpc.md]: superfluous dot.

File: ui/errors.go [hunk 4]
```diff
@@ -14,8 +14,9 @@ const (
 	ErrChannelNotFound
 	ErrDefailtGasPriceNotFound
 	ErrEmptyPassword
+	ErrMinConfirmationsNotFound
 	ErrOfferingNotFound
-	ErrSettingNotFound
+	ErrSetting
```
Comment (ababo) [inline ui/errors.go]: This error name is non-descriptive. ErrBadSettings, ErrFailedToDoSomethingWithSetting, etc.

**Pull request merged**